spec/factories/contacts.rb

```diff
@@ -16,5 +16,28 @@
   factory :contact do
     provider
     type { 'admin' }
```

Comment:
may as well

`admin_contact` instead of `type { 'admin' }`

Reply:
and done